cranelift/filetests/filetests/runtests/urem.clif

```diff
@@ -1,13 +1,11 @@
 test interpret
 test run
-target aarch64
-target s390x
-target x86_64
```

Comment:
These 3 lines test `urem`'s without the `avoid_div_traps` enabled and are not duplicates, although that is perhaps not very clear.

Reply:
I didn't know why these 3 line show deleted.
![image](https://user-images.githubusercontent.com/96557710/191631869-f42b56ea-5baa-4265-b20d-f25a294424a0.png)

Reply:
There were additional `target` lines above the `; Test these inputs without div traps, ...` line that have been removed in this PR. Could you please add them back.

Reply:
@bjorn3  sorry I didn't notice.